raddb/clients.conf

```diff
@@ -27,6 +27,16 @@
 #  the "ipaddr" or "ipv6addr" fields.  For compatibility, the 1.x
 #  format is still accepted.
 #
+client test {
```

Comment:
Nice we'll ship your clients and secrets with PacketFence. This way your setup can work with PF right out of the box